modules/cnf-about-nic-support-irq-affinity-setting.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: CONCEPT
+[id="about_nic_support_irq_affinity_setting_{context}"]
+= About NIC support of IRQ affinity setting
+
+Some IRQ controllers lack support for IRQ affinity setting and will always expose all online CPUs as the IRQ mask.
+
+The following list by no means exhaustive are examples of drivers and hardware that Red Hat are aware lack support for IRQ re-balancing:
+
+* Some RAID controller drivers, such as `megaraid_sas`
+* Many non-volatile memory express (NVMe) drivers
+* Some LAN on motherboard (LOM) network controllers
+
+[NOTE]
+====
+LOMs are actually NICs controlled by NIC drivers.
+The reason they do not support IRQ affinity setting, is probably associated with factors such as the type of processor, the IRQ controller, or the circuitry connections in the motherboard.
+====
```

Comment:
These note was a comment I made to explain why NIC drivers shouldn't be blamed for the IRQ affinity, but I see it a bit out of context here. As a user, why I need a clarification saying that a LOM are NICs?